set.mm

```diff
@@ -148815,8 +148817,8 @@ notation is used by Donald Knuth for iterated exponentiation (_Science_
       QWSFXAVEUSVFVGWRVISSHXBFAVJVKCWQKDVLVMVNWJKVOZBKVPGWOWMRENWPVQVRETVSAWAZT
       KQXJAEVTXKTKWBWCBWDBCKWEWFWGBFHWHWI $.
 
-    $( ` ``' G ` maps the size function's value to ` card ` .  (Contributed by
-       Paul Chapman, 22-Jun-2011.)  (Revised by Mario Carneiro,
+    $( The function ` ``' G ` maps the size function's value to ` card ` .
```

Comment:
Then "The converse of function ` G ` ..."

Reply:
Done. The reason of that edit was to avoid starting a sentence with a symbol.